quickmenu/arm9/source/main.cpp

```diff
@@ -1,2684 +1,2684 @@
-#include <nds.h>

```

Comment:
Sorry about the huge diff lol, couldn't stand the mixed newlines so I ran `dos2unix` on this file